src/System.IO.FileSystem.AccessControl/tests/FileSystemAclExtensionsTests.cs

```diff
@@ -16,24 +16,84 @@ public void GetAccessControl_DirectoryInfo_InvalidArguments()
         }
 
         [Fact]
+        public void GetAccessControl_DirectoryInfo_ReturnsValidObject()
+        {
+            using (var directory = new TempDirectory())
+            {
+                DirectoryInfo directoryInfo = new DirectoryInfo(directory.Path);
+
+                DirectorySecurity directorySecurity = FileSystemAclExtensions.GetAccessControl(directoryInfo);
```

Comment:
@ianhays Yes, they are all extensions.  The first test has the call using FileSystemAclExtensions so I continued the practice.

If you prefer, I can change the tests to call the extension off of the object instead.  Let me know what you think.

Thanks,

Michael